testing/ncids-css-testing/stories/components/nci-card/nci-card.testcases.group.without.description.stories.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Merge all test cases to one story file. Get rid of float. You can create a twig file here that calls the ncids-twig package. The params that are being passed to the twig should only be things that are actually a part of the card component